CONTRIBUTING.rst

```diff
@@ -6,6 +6,32 @@ help with implementing changes in Anaconda, please follow our
 `blog series <https://rhinstaller.wordpress.com/2019/10/11/anaconda-debugging-and-testing-part-1/>`_ or
 an `addon guide <http://rhinstaller.github.io/anaconda-addon-development-guide/index.html>`_ to create Anaconda addon.
 
+How to run make commands
+------------------------
+
+Anaconda has plenty of dependencies and because of that it's hard to set environment to
+with Anaconda properly. To get all the dependencies you are free to use script in the Anaconda
+repository. There are two ways how to approach.
+
+a) I want to have my host clean and run autotools from a container:
+
+``sudo dnf install toolbox``
+``toolbox create``
+``toolbox enter``
+
+Now, you are inside of the container.
+
+``sudo ./scripts/testing/install_dependencies.sh``
+``./autogen.sh && ./configure``
+
+You can exit the container by pressing `CTRL+d` and call make commands locally until any
+Makefile file is changed.
+
+b) To install the dependencies to my machine directly:
+
+``sudo ./scripts/testing/install_dependencies.sh``
+``./autogen.sh && ./configure``
```

Comment:
The commands are now exactly the same in toolbox and on the host (which is kinda the point). So how about instead of a) or b) you have a first paragraph which points out toolbox, and the second just says what to do regardless of the environment?

Reply:
Good idea, fixed.